packages/react/src/components/Layer/Layer.base.tsx

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore "react-portal-compat-context" uses v9 configs via path aliases
```

Comment:
wondering if we need this ? `react-portal-compat-context` is a dependency of this package so to be able to get proper types etc one needs to run `lage build --to` which will trigger build also for this v9 package, thus I assume this can be removed ?

Reply:
Yeah, you're totally correct 🐱 I made a change for N*, but then totally forgot about "build-me-first" behavior in `@fluentui/react`...